### DISCUSSION TITLE
INFO: Could not find files for the given pattern(s).

### DISCUSSION THREAD

**Discussion by cavan-mitchell** [smartcontractkit/full-blockchain-solidity-course-py#859] [Q&A] [answered]
cavan@DESKTOP-ILR3U0U MINGW64 ~/Documents/python practice/trading-bot
$ python deploy.py
INFO: Could not find files for the given pattern(s).

I have tried Bash & Powershell.

My concern is maybe I do not have contents in the correct folders. I'm relatively new to setting this up.

**Comment by cromewar** [comment, marked as answer]
Hello @cavan-mitchell 
INFO: Could not find files for the given pattern(s).
This is not a problem, is related to the translation between instructions on different command lines (powershell with cmd or bash with zsh etc). 
Sou have nothing to worry about.

**Reply by cavan-mitchell** [reply]
Thanks for your quick response.

My issue seemed to be more on web3 and dotenv properly. All the other issues seem to work itself out. 

I actually figured it out. web3 was not installing and I had to download the Microsoft Builder Tools that have the C++ related updates. After a did a restart, my error disappeared and was able to run my code.

**Reply by cromewar** [reply]
I'm glad is solved now, closing this thread

**Reply by n4n0b1t3** [reply]
I ran into the same non-issue. Everything works fine but I got this message "INFORMATION: Es konnten keine Dateien mit dem angegebenen Muster gefunden werden.". Since the information is localized, it is hard to post a topic or search for it here unless your system is English.
This actually appears when the only code in your file is
`from solcx import compile_standard`
